src/pascy/l2.py

```diff
@@ -12,15 +13,26 @@ class ArpLayer(Layer):
 
     @staticmethod
     def fields_info():
-        # TODO: Implement this :)
-        pass
+        return [UnsignedShort("hardware_type"),
+                UnsignedShort("protocol_type"),
+                UnsignedByte("hardware_len"),
+                UnsignedByte("protocol_len"),
+                UnsignedShort("operation"),
+                MacAddress("sender_hardware_addr"),
+                IPAddress("sender_protocol_addr"),
+                MacAddress("target_hardware_addr"),
+                IPAddress("target_protocol_addr")]
 
 
 class EthernetLayer(Layer):
     NAME = "Ethernet"
 
+    ARP_ETHER_TYPE = 0x0806
+    IPV4_ETHER_TYPE = 0x0800
+
     SUB_LAYERS = [
         [ArpLayer, "ether_type", 0x806],
+        [IpLayer, "ether_type", 0x800]
```

Comment:
As said before, adding a comma is recommended, as you may well see in this case 🙃